### ISSUE TITLE
Site para assistir anime

### ISSUE THREAD

**Issue by MarlonVitorinoo** [Mateus7G/crp-iframe-player#114]
Um dos melhores sites para assistir anime tanto dublado quanto legendado
https://betteranime.net/

**Comment by ghost** [comment]
> Um dos melhores sites para assistir anime tanto dublado quanto legendado https://betteranime.net/
Sim, e tbm também tem outro site muito bom que pega os animes diretamente da crunchyroll o https://puray.moe/ eu assisto nele e no better anime muito bom

**Comment by luannbr** [comment]
Aplicativo que recomendo tudo em fullhd é tudo séries, animes e filmes https://play.google.com/store/apps/details?id=com.psaappsa
